ptxboa/data/flh.csv

```diff
@@ -65,30 +62,6 @@ key,region,process_res,process_ely,process_deriv,process_flh,value,source
 "ARE,PV-FIX,SOEC-EL,NH3SYN,NH3-REC",ARE,PV-FIX,SOEC-EL,NH3SYN,NH3-REC,7008.0,öko
 "ARE,PV-FIX,SOEC-EL,NH3SYN,NH3SYN",ARE,PV-FIX,SOEC-EL,NH3SYN,NH3SYN,4809.0,PTX Atlas
 "ARE,PV-FIX,SOEC-EL,NH3SYN,SOEC-EL",ARE,PV-FIX,SOEC-EL,NH3SYN,SOEC-EL,4809.0,PTX Atlas
-"ARE,PV-TRK,AEL-EL,,H2-COMP",ARE,PV-TRK,AEL-EL,,H2-COMP,6657.599999999999,
```

Comment:
removing PTR-TRK is fine. But: do we use this flh data at all? I thought this has all been replaced by profile data and optimization results.

Reply:
yea, its still there for backwards compatibility